gcp/workers/alias/upstream_computation.py

```diff
@@ -156,12 +156,12 @@ def main():
   bugs = osv.Bug.query(
       ndb.OR(osv.Bug.upstream_raw > '', osv.Bug.upstream_raw < ''))
   bugs = {bug.db_id: bug for bug in bugs.iter()}
-  all_upstream_groups = osv.UpstreamGroup.query()
+  all_upstream_groups = osv.UpstreamGroup.query().fetch()
+  upstream_groups = {group.db_id: group for group in all_upstream_groups}
```

Comment:
nit: to avoid slight confusion with having two upstream group variables:
```suggestion
  upstream_groups = {group.db_id: group for group in osv.UpstreamGroup.query().iter()}
```